doc/api/http.markdown

```diff
@@ -25,11 +25,9 @@ HTTP API is very low-level. It deals with stream handling and message
 parsing only. It parses a message into headers and body but it does not
 parse the actual headers or the body.
 
-Defined headers that allow multiple values are concatenated with a `,`
-character, except for the `set-cookie` and `cookie` headers which are
-represented as an array of values.  Headers such as `content-length`
-which can only have a single value are parsed accordingly, and only a
-single value is represented on the parsed object.
+When headers are duplicated, either they're concatenated with `', '`,
+only the first one is taken, or they're presented as an array, depending
+on the header. See [message.headers][].
```

Comment:
LGTM but maybe link directly to `message.headers` without trying to describe the logic first?  Something like `See [message.headers][] for details on how duplicate headers are handled.`  That way it can't go out of sync.